test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -72,13 +73,32 @@ var _ = g.Describe("ScyllaCluster", func() {
 		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
 		defer cancel()
 
-		sc := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc := f.GetDefaultScyllaCluster()
 		sc.Spec.Repository = e.scyllaImageRepository
 		sc.Spec.Version = e.scyllaVersion
 		sc.Spec.Datacenter.Racks[0].Members = 3
 
+		supportsExposing, err := scyllafeatures.Supports(sc, scyllafeatures.ExposingScyllaClusterViaServiceOtherThanClusterIP)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		if !supportsExposing {
+			sc.Spec.ExposeOptions = &scyllav1.ExposeOptions{
+				NodeService: &scyllav1.NodeServiceTemplate{
+					Type: scyllav1.NodeServiceTypeClusterIP,
+				},
+				BroadcastOptions: &scyllav1.NodeBroadcastOptions{
+					Nodes: scyllav1.BroadcastOptions{
+						Type: scyllav1.BroadcastAddressTypeServiceClusterIP,
+					},
+					Clients: scyllav1.BroadcastOptions{
+						Type: scyllav1.BroadcastAddressTypeServiceClusterIP,
+					},
+				},
+			}
+		}
+
```

Comment:
if we ask the test suite to be run with a scylla version and some exposure type, this should better fail or skip, but probably fail

Reply:
I get the skip, but why would you want to fail it? Do you mean we should *expect* the test to fail for particular setups?

Reply:
if I run a suite with `--scyllacluster-nodes-broadcast-address-type=PodIP` and a accidentally a wrong scylla version I don't want it to silently pass - it needs to fail or we can have a green CI job and wrongly think it's working